src/components/menu-right/DotAppearanceModal.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="modal-card" data-test="dot-appearance-modal">
+    <header class="modal-card-head">
+      <p class="modal-card-title">Dot type {{ dotTypeIndex }} Appearance</p>
+    </header>
+
+    <section class="modal-card-body">
+      <svg viewBox="-1 -1 2 2" class="column">
+        <Dot
+          :key="`menu-right-dot-${dotTypeIndex}-preview`"
+          :dotTypeIndex="dotTypeIndex"
+          :labeled="false"
+        />
+      </svg>
+      <div class="column">
+        <b-field>
+          <b-checkbox v-model="dotAppearance.filled">Filled</b-checkbox>
+        </b-field>
+        <b-field>
+          <b-checkbox v-model="dotAppearance.fwSlash"
+            >Forward-slashed</b-checkbox
+          >
+        </b-field>
+        <b-field>
+          <b-checkbox v-model="dotAppearance.bwSlash"
+            >Backward-slashed</b-checkbox
+          >
+        </b-field>
+        <b-field>
+          <input type="color" v-model="dotAppearance.fill" /> Fill Color
+        </b-field>
```

Comment:
For all these `b-field` for the color input, can you use the `label` prop instead of putting the text next to the input and add the `horizontal` prop? See the docs: https://buefy.org/documentation/field
```suggestion
        <b-field label="Fill Color" horizontal>
          <input type="color" v-model="dotAppearance.fill" /> 
        </b-field>
```

You might be able to do this with the checkboxes as well... But if this looks weird, then disregard my comment!